libs/langgraph/langgraph/graph/branch.py

```diff
@@ -138,7 +142,15 @@ def run(
                 name=None,
                 trace=False,
                 func_accepts_config=True,
+            ),
+            list(
+                zip_longest(
+                    writer([e for e in self.ends.values() if e != END]),
```

Comment:
Why would the writer's output sequence length differ frmo the input sequence length?

Reply:
When you use `then` there is an extra write